about.html

```diff
@@ -0,0 +1,129 @@
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="css/about.css" />
+  </head>
+  <body>
+    <header>
+      <div class="social-header">
+        <ul class="social-nav">
+          <li class="top-social-icon">
+            <a href="#"><i class="fab fa-facebook"></i> </a>
+          </li>
+          <li class="top-social-icon">
+            <a href="#"> <i class="fab fa-twitter"></i></a>
+          </li>
+          <li class="top-social-icon">
+            <a href="#"> <i class="fab fa-youtube"></i></a>
+          </li>
+          <li class="top-social-icon"><a href="#">Login</a></li>
+          <li class="top-social-icon"><a href="#"> English</a></li>
+        </ul>
+      </div>
+      <div class="main-header" id="main-header">
+        <a href="index.html" class="logo">
+          <img src="images/logo.png" alt="Logo " />
+        </a>
+        <nav>
+          <ul class="main-nav hide-menu" id="main-nav">
+            <li>
+              <a href="index.html" class="nav-link">Home </a>
+            </li>
+            <li>
+              <a href="#" class="nav-link">Development</a>
+            </li>
+            <li>
+              <a href="#" class="nav-link">Design</a>
+            </li>
+            <li>
+              <a href="#" class="nav-link">IT & Software</a>
+            </li>
+            <li>
+              <a href="#" class="meetup">Buy A Course</a>
+            </li>
+          </ul>
+        </nav>
+        <div id="hamburger" class="menu">
+          <div class="line-1 bar"></div>
+          <div class="contact bar"></div>
+          <div class="line-3 bar"></div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="about-head-line">
+        <h2 class="title-secondary">"Hello! Curious Fellows"</h2>
+        <h1 class="motto">We are bringing Revolution by Evolution to Web</h1>
+        <p class="text-description">
+          we will take your skills to a professional level, take the first
+            step by trying out one of numerous free, self-guided online web
+            development courses  that cover everything from basic programming 
+            introductions to development best practices. Stay Tune and Let Start a wonderful journey. 
+        </p>
+        <p class="contact">
+          Please Contact Us if you have any query.
+        </p>
+        <a href="#" class="link">admin@webevolution.com</a>
+      </section>
+      <section class="container logo-container">
+        <h3 class="title-3"> Web Evolution Idea 2022</h3>
+        <p class="text-3"> The idea of web courese was taken in a seminar about the current state of web and the content available</p>
+          <div class="idea-logo">
+            <img src="images/logo.png" alt="logo" />
+          </div>
+        <div class="our-experience">
+          <h3 class="title-3">Our Success Story</h3>
```

Comment:
- [ ] The two images for this section can be made wider so as to make the site look more like the given design in mobile view. Please do so.

- [ ]  The font for the inner text of these images has not been applied, kindly apply the correct font so as to beautify your site even more.

Your design
![image](https://user-images.githubusercontent.com/81932478/169672688-1f40bbd5-1a0a-4a3a-93aa-f4bfa10c1ba9.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/169672699-9e052db1-1a59-49f9-bfca-3607744a1885.png)

The margin on the left and right of the images is too much it makes the images small in your design and far from the border of the main container.

- [ ] Also the top text should be bold and centered while the bottom text should have a normal font weight. Kindly fix this as well.

Reply:
Resolved